docs/markdown/Syntax.md

```diff
@@ -527,6 +527,54 @@ executable('progname', 'prog.c',
 
 Attempting to do this causes Meson to immediately exit with an error.
 
+### Argument flattening
+
+Argument flattening is a Meson feature that aims to simplify using
+methods and functions. For functiosn where this feature is active,
```

Comment:
"For functiosn" -> "For functions"

Reply:
Thanks for finding those typos. I really should have used a spellchecker before opening this PR :)